src/server.h

```diff
@@ -2345,6 +2346,30 @@ typedef struct {
     unsigned char *lpi; /* listpack iterator */
 } setTypeIterator;
 
+/* Structure for DB iterator that allows iterating across multiple slot specific dictionaries in cluster mode. */
+typedef struct dbIterator {
+    redisDb *db;
+    int index;
+} dbIterator;
+
+/* DB iterator specific functions */
+/* Returns DB iterator that can be used to iterate through sub-dictionaries.
+ * Primary database contains only one dictionary when node runs without cluster mode,
+ * or 16k dictionaries (one per slot) when node runs with cluster mode enabled. */
+dbIterator *dbGetIterator(redisDb *db);
```

Comment:
Nice!

Though in Redis, these documentation comments are normally in the `.c` files.

Reply:
Okay, will move.